cdc/sink/dispatcher/event_router.go

```diff
@@ -26,6 +26,17 @@ import (
 	"go.uber.org/zap"
 )
 
+// DDLDispatchRule is the dispatch rule for DDL event.
+type DDLDispatchRule int
+
+const (
+	// Broadcast means the DDL event will be broadcast to all the partitions.
+	Broadcast DDLDispatchRule = -1
```

Comment:
Maybe `PartitionAll` is a better name to keep consistent with `PartitionZero`.

Reply:
Cool!